### PR TITLE
[kvm] Use run_tests.sh script in sonic-mgmt to run PR tests

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

Uses the `run_test.sh` script from `sonic-mgmt` to run the pytests to catch regressions in `run_test.sh` during PR testing. Next step is to use the "device_type=vs" marker or something similar to run the tests rather than using a hard-coded list.

### CODE REVIEW

**Comment by daall** [comment]
https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/jenkins_testing/job/sonic-mgmt-test/11/

File: scripts/vs/buildimage-vs-image/runtest.sh [hunk 1]
```diff
@@ -40,67 +22,61 @@ export ANSIBLE_LIBRARY=/data/sonic-mgmt/ansible/library/
...
+    -e --disable_loganalyzer
+"
 
-# Check testbed health
 cd /data/sonic-mgmt/tests
-rm -rf logs results
+rm -rf logs
 mkdir -p logs
-mkdir -p results
-py.test $PYTEST_COMMON_OPTS --log-file logs/test_nbr_health.log --junitxml=results/tr.xml test_nbr_health.py
```
Comment (lguohan) [inline scripts/vs/buildimage-vs-image/runtest.sh:58]: do we check nbr health now if this is removed?

File: scripts/vs/buildimage-vs-image/runtest.sh [hunk 2]
```diff
@@ -40,67 +22,61 @@ export ANSIBLE_LIBRARY=/data/sonic-mgmt/ansible/library/
...
-
-# Run anounce route test case in order to populate BGP route
-py.test $PYTEST_COMMON_OPTS --log-file logs/test_announce_routes.log --junitxml=results/tr.xml test_announce_routes.py
```
Comment (lguohan) [inline scripts/vs/buildimage-vs-image/runtest.sh:61]: do we announce route now?

Comment (daall) [reply scripts/vs/buildimage-vs-image/runtest.sh:61]: Yep, it is part of the "pretest" steps!

File: scripts/vs/buildimage-vs-image/runtest.sh (hunk 1 above)
Comment (daall) [reply scripts/vs/buildimage-vs-image/runtest.sh:58]: Yep, also part of the pretest steps! @yxieca can confirm the exact behavior.

**Pull request merged**